rhizome/postgres/bin/metrics-collector

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require "time"
+require "json"
+require "shellwords"
+
+require_relative "../../common/lib/util"
+
+if ARGV.count != 2
+  fail "Wrong number of arguments. Expected 2, Given #{ARGV.count}"
+end
+
+metrics_dir = ARGV[0]
+q_metrics_dir = metrics_dir.shellescape
+metrics_config_path = ARGV[1]
```

Comment:
I feel like these 2 parameters can be hard-coded.

Reply:
Based on the conversation in other threads, I think it makes sense to keep metrics_dir as parameter. That said, metrics_config_path can still be hard-coded.